app/static/src/app/store/run/mutations.ts

```diff
@@ -38,5 +39,9 @@ export const mutations: MutationTree<RunState> = {
         state.endTime = payload;
         const prevEndTime = state.result?.inputs?.endTime ? state.result.inputs.endTime : -1;
         state.runRequired = payload > prevEndTime;
+    },
+
+    [RunMutation.SetDefaultEndTime](state: RunState, payload: number) {
```

Comment:
I'm not clear on why this mutation is needed rather than just going through `SetEndTime`? It looks like `SetDefaultEndTime` is only called on app startup when `state.result` will be `null` so as far as I can see it has the same practical effect?

Reply:
Agreed